npm-react/react.js

```diff
@@ -1 +1,61 @@
-module.exports = require('./lib/React');
+'use strict';
+
+var React = require('./lib/React');
+var ReactDOMClient = require('./lib/ReactDOMClient');
+var ReactDOMServer = require('./lib/ReactDOMServer');
+
+var assign = require('./lib/Object.assign');
+var warning = require('./lib/warning');
+
+
+// We want to warn once when any of these methods are used.
+if (process.env.NODE_ENV !== 'production') {
+  var warnedForReactDOMClient = false;
+  var warnedForReactDOMServer = false;
+  var wrappedDOMClient = {};
+  var wrappedDOMServer = {};
+  var domClientKeys = Object.keys(ReactDOMClient);
+  var domServerKeys = Object.keys(ReactDOMServer);
+  domClientKeys.forEach(function(name) {
+    wrappedDOMClient[name] = function() {
+      if (!warnedForReactDOMClient) {
+        warning(
+          false,
+          '`%s` have been deprecated from the `react` package and will be ' +
+          'removed in a future version. See http://fb.me/react-isomorphic ' +
+          'for more details.',
+          domClientKeys.join('`, `')
+        );
```

Comment:
I was actually thinking of deleting that. It's going to warn once per method call, which I thought was too much. `deprecated` is also pretty inflexible.

I could make it

```js
deprecated('require("react")', name, 'require("react-dom")', name, ReactDOMClient, ReactDOMClient[name])
```

But then we couldn't customize the message with an fb.me link (which I think we should have).